sdk/storage/azure-storage-blob/azure/storage/blob/_shared/parser.py

```diff
@@ -18,3 +18,10 @@ def _str(value):
 
 def _to_utc_datetime(value):
     return value.strftime('%Y-%m-%dT%H:%M:%SZ')
+
+
+def get_empty_chunk(chunk_size):
+    empty_chunk = b''
+    for i in range(0, chunk_size):  # pylint:disable=unused-variable
+        empty_chunk += b'\x00'
```

Comment:
>empty_chunk += b'\x00' [](start = 8, length = 22)

Much like string concatenations, this can be very inefficient, as the append operations keep building new byte arrays and copying old contents to them.